Makefile

```diff
@@ -7,20 +7,19 @@ export GO15VENDOREXPERIMENT=1
 VERSION := 0.0.1-$(shell date "+%Y%m%d%H%M%S")
 LDFLAGS := "-s -X main.version=${VERSION}"
 BINDIR := ./rootfs/bin
-DEV_REGISTRY ?= $(eval docker-machine ip deis):5000
+DEV_REGISTRY ?= $(docker-machine ip deis):5000
```

Comment:
Shouldn't this be `$(shell docker-machine...)`? IIRC, there are cases where dm may need env info.

https://www.gnu.org/software/make/manual/html_node/Shell-Function.html

I thought I commented on this before, but I can't find that comment.